### PR TITLE
Issue20

### PR DESCRIPTION
**Pull request by kellobri**
Draft 2.0 ready for review

### CODE REVIEW

File: docs/easyingest/easyingest_introduction.rst [hunk 1]
```diff
@@ -1,4 +1,49 @@
...
+At this step you’ll be prompted to select a destination for the data you’ve configured. EasyIngest offers different destination options based on the source you’ve selected.
+There are two types of destinations to choose from: 1) Platform destinations (Stacks), and 2) Outside destinations (Cloud storage solutions like AWS S3).
+
+**4. Configure your Ingest Destination** - Set up access to the destination
+
+The final step is to configure your EasyIngest job with any necessary information about the destination selected in Step 3.
+As with source configuration, the information collected in this step will differ based on the destination. The form is again designed to require only the most minimal input interaction.
+
+
+**Running EasyIngest** - Next Steps:
```
Comment (audevans) [inline docs/easyingest/easyingest_introduction.rst]: Remove colon

File: docs/easyingest/easyingest_introduction.rst [hunk 2]
```diff
@@ -1,4 +1,49 @@
...
+
+After you submit the ingest job at the end of Step 4, the EasyIngest process will spawn a page where you may observe it running tasks.
+
+Once the status of your job reaches the green *running* state, you should be able to access/view/connect to your data at the appropriate destination location.
```
Comment (audevans) [inline docs/easyingest/easyingest_introduction.rst]: Probably need some type of 'warm' comment here like...  You and your data are now up and running!  Or something cheesy like that.

File: docs/easyingest/easyingest_introduction.rst [hunk 3]
```diff
@@ -1,4 +1,49 @@
...
+
+In the first step, you’ll choose your data source. We offer a variety of open source and alternative data sources, as well as free-form cloud sources like AWS S3 and Redshift. Please note that you must have a Space running on the platform to start any of the ingest sources.
+
+If you ever need an ingest data source that isn’t provided, please contact us at info@b23.io. User feature requests and feedback are a major driving force in shaping the future of our platform.
+
+**2. Define your Ingest Source** - Set up access to the data
+
+Based on the ingest source you chose in Step 1, defining the data source will require different simple inputs. All the source setup forms are designed to be intuitive and easy while also offering as much feature flexibility as possible.
+
+Configuration Considerations:
```
Comment (audevans) [inline docs/easyingest/easyingest_introduction.rst]: For the following list of three, may want to title the considerations:
Alternative/Open Source Data:
Redshift Clusters:
S3 Data:

Comment (kellobri) [inline docs/easyingest/easyingest_introduction.rst]: I made some updates - let me know if this is what you had in mind

File: docs/easyingest/easyingest_introduction.rst [hunk 4]
```diff
@@ -1,4 +1,51 @@
-EasyIngest with Stackspace
---------------------------
+EasyIngest with B23 Data Platform
+---------------------------------
 
-EasyIngest is ready to go in Stackspace, and its use is straight forward and simple to use.  We are still working on the documentation although, so if you would like to use EasyIngest and have questions about how to use it, please email us at info@stackspace.io.
+EasyIngest: Secure Data Access and Delivery in the Cloud
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enterprise data stored in the cloud can be quickly and securely accessed on the B23 data platform via our EasyIngest feature. With EasyIngest you can expect:
```
Comment (audevans) [inline docs/easyingest/easyingest_introduction.rst]: Capitalize: B23 Data Platform

File: docs/easyingest/easyingest_introduction.rst [hunk 5]
```diff
@@ -1,4 +1,51 @@
...
+
+- **Flexible Data Ingest Solutions**: B23 EasyIngest can support bulk, streaming and connection-based ingest methods for a variety of cloud data sources
+
+- **Enterprise-ready Data Security**: In addition to the security features of a B23 data platform virtual private cloud (Space), B23 [the company] never stores, copies or has the means to access data you send through EasyIngest
```
Comment (audevans) [inline docs/easyingest/easyingest_introduction.rst]: Capitalize: B23 Data Platform

File: docs/easyingest/easyingest_introduction.rst (hunk 3 above)
Comment (audevans) [inline docs/easyingest/easyingest_introduction.rst]: Sweet!

**Comment by kellobri** [comment]
@bjkolly The Easy Ingest Intro is ready for your review

**Pull request merged**